tensorflow/java/src/gen/perl/tftypes-runall.pl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/perl
```

Comment:
My concern is that the code generation process is more layers for someone who needs to make a change to sift through. Removing option handling form `tftypes.pl` would make it simpler for both a reader and an editor IMHO. One could always use source control to restore unexpected changes to generated files. 

So I believe we agree in keeping `tftypes.pl` as simple as possible, we just have a slight disagreement on what makes it so :). That said, happy to defer to you for now (and will possibly revisit when redoing this in Python)